### ISSUE TITLE
Transcode = Auto choose wrong method (not sure why)

### ISSUE THREAD

**Issue by Fredde87** [iBaa/PlexConnect#407]
Hey guys,

So I ran into a bit of a problem with one of my Apple TV's. I was doing some testing in regards to a separate issue I raised.

Basically I have a large library of iTunes movies which I use PlexConnect to serve to my Apple TVs together with my other videos files. Since these are DRM protected they can only play via DirectPlay. And this has worked for years and still does on 15 out of my 16 Apple TVs. But on one of them I was doing some testing with Subtitles and Audio Language and it stopped playing my DRM content. I figured the Apple TV must have remembered some selection I did so I reset it to factory default and installed it from scratch. But it still won't play the content.

I then figured out that even though I have Transcoding set to Auto, for some reason it tries to Transcode it rather than DirectPlay it. But all the other Apple TVs are set to Auto as well and they correctly choose to DirectPlay it.

I had a look in my ATVSettings.cfg and there is nothing odd for that Apple TV that would cause it. Is there some sort of cache somewhere else (in PlexConnect or Plex) which might remembered that it tried to transcode files for this client in the past (due to me playing with subtitles and audio languages) which might explain why it is trying to transcode in Auto mode to this one client but not the rest?

**Comment by Fredde87** [comment]
Sorry forgot to mention that maybe this has something to do with dolby digital as well? When I set transcoding to DirectPlay then the movie plays but I got load static noise so I switched dolby digital off and I get normal audio. And since all of this started when I was playing with the audio language I get a feeling that it has something to do with it. (normally dolby digital is on and works for me)
